src/app/src/main/kotlin/org/imperial/mrc/hint/HintApplication.kt

```diff
@@ -4,7 +4,7 @@ import org.springframework.boot.SpringApplication
 import org.springframework.boot.autoconfigure.SpringBootApplication
 
 @SpringBootApplication
-class HintApplication
+open class HintApplication
```

Comment:
Springboot started complaining that Configuration and Bean classes and methods have to be non-final. According to the documentation that is correct, but I'm not sure why it wasn't a problem before?

Reply:
How bizarre that it wasn't complaining before? Did we have a plugin that wasn't being activated somehow?